### ISSUE TITLE
[FL autosuggest prototype] <input> text doesn’t enlarge when browser text settings are set to enlarge

### ISSUE THREAD

**Issue by qjac** [department-of-veterans-affairs/va.gov-cms#22512]
## Describe the defect
Found as part of [ADE accessibility audit](https://github.com/department-of-veterans-affairs/va.gov-team/issues/120537#issuecomment-3347814120) of the location search box, Facility type select, custom select, and autosuggest fields on the [autosuggest prototype](https://staging.va.gov/find-locations/).

> 
> **Issue: <input> text doesn’t enlarge**
> **Field:** Location search box, autosuggest
> **Expected severity level:** 1
> **Expected experience standard:** User can perceive all meaningful elements
> **Description:** Text within your custom <input> components doesn’t resize when the browser font size is increased:
> <img width="353" height="412" alt="Image" src="https://github.com/user-attachments/assets/05ddb1cc-89b8-4bca-9f0a-beae2afa59c2" />
> **How to recreate:** Using Chrome, go to the Settings, search for “font,” and increase the font size to “Very Large.” Then, try typing into the <input> fields.
> **Recommendation:** This works properly for VADS va-text-input components. But assuming you can’t switch to that, you should set the font size on these inputs to 1rem. (It’s inheriting 16px from the cascade.)

[Location input and services autosuggest are custom comboxes](https://dsva.slack.com/archives/C0FQSS30V/p1759523788832689?thread_ts=1759522358.548369&cid=C0FQSS30V) and are therefore not inheriting expected styling from VADS inputs. We want to match how [va-text-input](https://design.va.gov/components/form/text-input) handles text: it inherits `font-size: 1.06rem` from `.usa-input`. Ours is currently inheriting an absolute value and not enlarging when settings are changed.

<img width="1714" height="297" alt="Code with font-size: 16px set." src="https://github.com/user-attachments/assets/0cc1a7bd-4f78-417e-8dee-831d80b125ce" />

## Recommendation
- Put `.usa-input` on the location and service type inputs to inherit USWDS text styles. Or if that's not an option, set the input text size to 1.06rem to match USWDS/VADS text sizing.

[VADS mock form showing desired implementation](https://staging.va.gov/mock-simple-forms-patterns/text-input)
[VADS combo box Storybook component](https://design.va.gov/storybook/?path=/docs/uswds-va-combo-box--docs)

## To Reproduce
1. Using Chrome, go to the Settings, search for “font,” and increase the font size to “Very Large.” 
2. Go to [autosuggest prototype](https://staging.va.gov/find-locations/)
3. Type into the <input> fields (location and VA health > services type).
4. See that the text typed into the input is not scaled like the surrounding text is.
5. Using devtools, you can see that font size for the input is 16px.

## AC / Expected behavior
- [ ] GIVEN a user is on facility locator prototype AND has set browser text to be enlarged, WHEN the user types into an input, THEN the typed text is enlarged.
- [ ] Requires accessibility review

 ## Refinement questions
- I tried adding .usa-input to our input using devtools and it did not bring in expected USWDS styles. When I remove the .usa-input from the inputs in the mock form, I see a dramatic difference in styling. Why is this page not pulling in styles like the form demo page is?

**Comment by qjac** [comment]
@carlyMhayter @r-bartlett-gsa, confirmed in Chrome that the options in the dropdown adjust sizing based on browser font settings. 🎉
